File: Leetcode/code/28.py
def strStr(haystack: str, needle: str) -> int:
    # 들어있으면 인덱스를 리턴하고 
    if needle in haystack:

        print(haystack.index(needle))

    else: return -1

strStr("hello", "ll")
    # 들어있지 않으면 -1 을 리턴하는 것
    

File: leetcode/28.js
const strStr = (haystack, needle) => {
    // Checking to prevent unecessary verification
    if (needle === '') return 0;
    if (haystack.length < needle.length) return -1;
    
    // Do the checking
    for (let i = 0; i < haystack.length - needle.length + 1; i+= 1) {
        // In case the character matches with the first in the needle
        if (haystack.charAt(i) === needle.charAt(0)) {
            // Check all the other characters
            let matches = true;
            for (let j = 1; j < needle.length; j++) {
                if (haystack.charAt(i + j) !== needle.charAt(j)) matches = false;
            }
            // Matches
            if (matches) return i;
        }
    }
    
    // Did not find
    return -1;
};